plugins/nanospell/plugin.js

```diff
@@ -65,6 +65,9 @@
 		var startNode = range.startContainer;
 		var endNode = range.endContainer;
 
+		var self = this;
+		this.hitNestedBlock = false;
```

Comment:
I think the only place it actually matters is `isRootBlockTextNode` which is used as a callback, though.  not sure.  thoughts?